native/src/routes/jpal-tracking/JpalTracking.js

```diff
@@ -0,0 +1,138 @@
+// @flow
+
+import React, { useEffect, useState } from 'react'
+
+import type { ThemeType } from 'build-configs/ThemeType'
+import { Switch, Text, TextInput, View } from 'react-native'
+import { useTranslation } from 'react-i18next'
+import AppSettings, { defaultSettings } from '../../modules/settings/AppSettings'
+import type { NavigationPropType, RoutePropType } from '../../modules/app/constants/NavigationTypes'
+import type { JpalTrackingRouteType } from 'api-client'
+import styled from 'styled-components/native'
+import LayoutContainer from '../../modules/layout/containers/LayoutContainer'
+import withTheme from '../../modules/theme/hocs/withTheme'
+import Caption from '../../modules/common/components/Caption'
+
+const ThemedText = styled.Text`
+  display: flex;
+  text-align: left;
+  color: ${props => props.theme.colors.textColor};
+  font-family: ${props => props.theme.fonts.native.decorativeFontRegular};
+  padding: 10px 0;
+`
+
+const ErrorText = styled.Text`
+  color: red;
+  font-weight: bold;
+  padding: 10px 0;
+`
+
+const DescriptionContainer = styled.TouchableOpacity`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 15px 0 5px;
+`
+
+const Input = styled(TextInput)`
+  padding: 15px;
+  border-width: 1px;
+  border-color: ${props => props.theme.colors.themeColor};
+  text-align-vertical: top;
+  height: 50px;
+`
+
+export type PropsType = {|
+  theme: ThemeType,
+  route: RoutePropType<JpalTrackingRouteType>,
+  navigation: NavigationPropType<JpalTrackingRouteType>
+|}
+
+const errorDisplayTime = 5000
+
+const appSettings = new AppSettings()
+
+const JpalTracking = (props: PropsType) => {
+  const [settings, setSettings] = useState(defaultSettings)
+  const [settingsLoaded, setSettingsLoaded] = useState(false)
```

Comment:
As mentioned in the coworking: I think those could be omitted :)

Reply:
We talked abour settingsUpdated, i removed it. But i think we should keep settingsLoaded, otherwise the page is rendered, when the date is not present yet and then (after a short time), when the data is present, the switch and the textfield get updated, what looks weird